### DISCUSSION TITLE
第2节课作业(9班)

### DISCUSSION THREAD

**Discussion by RangeKing** [InternLM/Tutorial#101] [作业]
## 基础作业：

- 使用 InternLM-Chat-7B 模型生成 300 字的小故事（需截图）。

- 熟悉 hugging face 下载功能，使用 huggingface_hub python 包，下载 InternLM-20B 的 config.json 文件到本地（需截图下载过程）。

## 进阶作业（可选做）

- 完成浦语·灵笔的图文理解及创作部署（需截图）

- 完成 Lagent 工具调用 Demo 创作部署（需截图）

## 整体实训营项目：

时间周期：即日起致课程结束

即日开始可以在班级群中随机组队（5人及以内）完成一个大作业项目，一些可提供的选题如下：

1. 人情世故大模型：一个帮助用户撰写新年祝福文案的人情事故大模型

2. 中小学数学大模型：一个拥有一定数学解题能力的大模型

3. 心理大模型：一个治愈的心理大模型

4. 工具调用类项目：结合Lagent构建数据集训练InternLM模型，支持对MMYOLO等工具的调用

5. 其他基于书生·浦语工具链的小项目都在范围内，欢迎大家充分发挥想象力。

**Comment by liuxianyi** [comment]
[趣味Demo](https://liuxianyi.notion.site/Demo-34ea43b2fbf84250af5c9406b7eaacc9?pvs=4)（作业在笔记中）

**Comment by xiaomile** [comment]
[作业1](https://github.com/xiaomile/InternLM-homework/tree/main/%E4%BD%9C%E4%B8%9A1)

**Comment by li-chuanwei** [comment]
https://www.yuque.com/epiphany-erkzt/epcy7g/tflm2iomnifydgbk?singleDoc# 《作业2》

**Comment by GingerNg** [comment]
https://note.youdao.com/s/Bw7r19ry

**Reply by blissSeven** [reply]
2轻松玩转书生·浦语大模型趣味Demo-作业

https://github.com/blissSeven/knowledge/blob/master/LLM/2%20%E8%BD%BB%E6%9D%BE%E7%8E%A9%E8%BD%AC%E4%B9%A6%E7%94%9F%C2%B7%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B%E8%B6%A3%E5%91%B3Demo-%E4%BD%9C%E4%B8%9A.md

**Comment by ShelterWFF** [comment]
class2: http://t.csdnimg.cn/NKD1b

**Comment by nxznm** [comment]
第二课：https://zhuanlan.zhihu.com/p/676438847

**Comment by hzz12138** [comment]
class2：https://note.youdao.com/s/TESsBjYa

**Comment by KJ-Huang-sz** [comment]
第二次作业：https://www.bilibili.com/read/cv29101782/?jump_opus=1

**Comment by leonnapsun** [comment]
第二课作业：
https://zhuanlan.zhihu.com/p/676516608

**Comment by li4692625** [comment]
https://blog.csdn.net/li4692625/article/details/135442811

**Comment by i-am-so-lucky-today** [comment]
第二课作业：
https://blog.csdn.net/weixin_49534890/article/details/135443300?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135443300%22%2C%22source%22%3A%22weixin_49534890%22%7D

**Comment by SaigeRing** [comment]
https://zhuanlan.zhihu.com/p/676492843

**Comment by xingyuan3** [comment]
https://www.mubu.com/doc/Uef-RoqCRd

**Comment by wangxuejiao** [comment]
第二节作业：https://zhuanlan.zhihu.com/p/676586122

**Comment by nini0919** [comment]
第2节课作业:https://www.notion.so/1-Demo-2cf405ebf00c44f9befae88a4a3ad2fb?pvs=4

**Comment by hahahamamama** [comment]
第二课作业：
https://github.com/hahahamamama/InternLM/blob/main/Work1.md

**Comment by haizeigh** [comment]
第二课作业
https://note.youdao.com/s/FhzLD7L

**Comment by AIzealotwu** [comment]
https://blog.csdn.net/m0_72806612/article/details/135482625?spm=1001.2014.3001.5502

**Comment by dragonzzl** [comment]
作业2：https://github.com/dragonzzl/works_internlm/blob/main/works/work02.md

**Comment by RangeKing** [comment]
https://rangeking.github.io/#/page/2.%20internlm%20demo
